### DISCUSSION TITLE
restore all settings

### DISCUSSION THREAD

**Discussion by soyeric128** [databendlabs/databend#7735] [Q&A] [answered]
Hi there, 

Since we have a bunch of settings through which users can customize Databend to fit their business, even they can show the current values by SHOW SETTINGS, is there a way to reset all the settings to their original values as users may change them back and forth?  

That is, instead of setting one by one by their default value, can we have a command like reset all settings?

**Comment by Xuanwo** [comment]
cc @TCeason for ideas.

**Comment by TCeason** [comment, marked as answer]
Maybe can ref this issue. https://github.com/datafuselabs/databend/issues/8099
